bgpd/bgp_bfd.c

```diff
@@ -282,7 +282,8 @@ static void bgp_bfd_peer_status_update(struct peer *peer, int status,
 		return;
 
 	old_status = bfd_info->status;
-	bfd_info->status = status;
+	BFD_SET_CLIENT_STATUS ((bfd_info->status), status);
```

Comment:
Please, for this and all other occurences, don't use space before opening the macro bracket and don't use the extra parentheses for the variable.

Example:

```c
BFD_SET_CLIENT_STATUS(bfd_info->status, status);
```

Reply:
Done